### ISSUE TITLE
New Topics needed

### ISSUE THREAD

**Issue by Muthukamalan** [animatedai/animatedai.github.io#3]
Hi @animatedai ,

Your video is super intuitive. You own my respect. If would be great if you  add batch norm, layer norm, group norm, instance norm like that

**Comment by animatedai** [comment]
Hi @Muthukamalan,

Thanks for the feedback. Are there any of those normalization techniques that you're more interested in seeing than the others? And are you looking for an explanation of each technique on its own or are you looking for more of a comparison between the techniques (e.g., strengths/weaknesses, when to use one vs. another, etc...)?

**Comment by Muthukamalan** [comment]
Thanks for your response @animatedai 

Explanation of following techniques and how it'll updating it's hyperparameter during training
- Layer Norm (LN)
- Batch Norm (BN)
- Instance Norm (IN)
- Group Norm (GN)

I see a lot of stackoverflow and reddit response quoting saying use LN for NLP, BN for images (in LN paper, Hinton quotes same). but I like to get power to visualize like you did it in your previous videos ❤️
